test/OAuth2ManagerTests/packages.config

```diff
@@ -1,6 +1,8 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.Taef" version="10.94.240624002" targetFramework="native" />
-  <package id="Microsoft.Windows.CppWinRT" version="2.0.240915.1" targetFramework="native" />
+  <package id="Microsoft.Taef" version="10.95.240918004" targetFramework="native" />
+  <package id="Microsoft.Windows.CppWinRT" version="2.0.230706.1" targetFramework="native" />
+  <package id="Microsoft.Taef" version="10.95.240918004" targetFramework="native" />
```

Comment:
There are two sets of these.

Reply:
Yup. There _were_. Due to a bad merge. Fixed in recent commit yesterday